server/api/shared.py

```diff
@@ -676,6 +690,19 @@ def compute_percentage_of_jurisdiction_ballots_selected(
     return sample_batches
 
 
+def compute_sample_batches(
+    election: Election,
+    round_num: int,
+    contest_sample_sizes: List[Tuple[Contest, SampleSize]],
+) -> List[BatchDraw]:
+    sample_batches: List[BatchDraw] = []
+    for contest, sample_size in contest_sample_sizes:
+        sample_batches += compute_sample_batches_for_contest(
+            election, round_num, contest, sample_size
+        )
+    return sample_batches
```

Comment:
consider a list comprehension here

Reply:
Went ahead and made the switch, just noting that I sometimes find nested list comprehensions to be hard to grok: https://github.com/votingworks/arlo/pull/1884/commits/f7c59dc11649a596050f13a54ba2c06d6c7d1aad

Reply:
Fair, I also find the order of statements in a nested comprehension confusing. My brain always wants it to be the other way around